config/locales/two_factor_authentication/en.yml

```
@@ -38,6 +38,8 @@ en:
     invalid_piv_cac: That PIV/CAC didn’t work. Make sure it’s the right PIV/CAC for
       this account. If it is, there may be a problem with your PIV/CAC, PIN, or something
       went wrong on our end. Try again or choose another authentication method.
+    lockout_information: Keep this informations safe. You will be locked out and have
```

I see it's in the ticket and the original Figma design, but can we confirm that this is supposed to be "informations" and not "information" ? It seems to read better to me as the singular 🤔

---

@aduth Great catch. I don't think "informations" is correct (and maybe not even a word).

```suggestion
    lockout_information: Keep this information safe. You will be locked out and have
```